src/logistration/RegistrationFailure.jsx

```diff
@@ -2,48 +2,61 @@ import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from '@edx/frontend-platform/i18n';
 import { Alert } from '@edx/paragon';
+import { INTERNAL_SERVER_ERROR } from './data/constants';
 
-const hasNoErrors = (userErrors) => (
-  userErrors.every((errorList) => (!errorList[0]))
+const showAlert = (userErrors) => (
+  <Alert variant="danger">
+    <Alert.Heading className="text-danger">
+      <FormattedMessage
+        id="logistration.registration.request.failure.header.description.message"
+        defaultMessage="We couldn't create your account."
+        description="error message when registration failure."
+      />
```

Comment:
We should add these string to messages.jsx 
nit: Also the id is too long, `registration.request.failure.header` is sufficient